doc/userguide/install.rst

```diff
@@ -147,27 +147,81 @@ For Debian 10 (buster), for instance, run the following as ``root``::
     apt-get update
     apt-get install suricata -t buster-backports
 
-Fedora
-^^^^^^
+CentOS, AlmaLinux, RockyLinux, Fedora, etc
```

Comment:
Large enough section go have its own page? I'm thinking so. I'd like to see similar for Debian packages as well.

Reply:
Wouldn't hurt, imo.